dump/dump.js

```diff
@@ -85,13 +85,23 @@ function onPacket(packet, params) {
       frame.zcl = zclData;
       dumpFrame(label, frame, DEBUG_frameDetail);
     });
-    dumpFrame(label, frame, DEBUG_frameDetail);
   } else {
     dumpFrame(label, frame, DEBUG_frameDetail);
   }
 }
 
+function doPacket(packetStr, params) {
+  // Strip spaces from packetStr
+  packetStr = packetStr.replace(/\s+/g, '');
+  const packet = new Buffer(packetStr, 'hex');
+  params.timeStamp = 0;
+  params.frameNum = 1;
+  onPacket(packet, params);
+}
+
 const optionsDefs = [
+  {name: 'readPacket', alias: 'p', type: String},
```

Comment:
The read alias makes me sad.

Reply:
Yeah - i was already using -r for the raw option and since I use -r for raw in a few other programs I was reluctant to use it instead.